packages/odyssey-react-mui/src/Dialog.tsx

```diff
@@ -85,8 +85,11 @@ const Dialog = ({
         {title}
         <Button
           variant="floating"
+          size="small"
           onClick={onClose}
           startIcon={<CloseIcon />}
+          ariaLabel="Close dialog"
+          text=""
```

Comment:
Do we have to pass `text=""`? Can't just pass an icon and leave `text` undefined?

This might be a failure in the type system.

Reply:
We didn't previously, but I noticed it wasn't picking up the required styles without it.

I think this has to do with our changes relating to `text` vs `children` and no one noticed the styles broke because the surface area is so small.